### ISSUE TITLE
.davmail.properties should be created with 600 permissions

### ISSUE THREAD

**Issue by rsekman** [mguessan/davmail#254]
`.davmail.properties` can contain, in plaintext, proxy passwords an OAuth tokens that should be kept secret. It should not be world-readable. While on Unix-like systems this is easily fixed with `chmod 600` this should not be the responsibility of the end-user. `davmail` should create this file with the appropriate permissions from the start by calling `umask()`.

**Comment by mguessan** [comment]
Makes sense, now that we dropped support for older java versions we can use nio API.
=> fix available in trunk, trying to set file readable by user on initial file creation
